owlbot.py

```diff
@@ -75,48 +78,49 @@
 
 
 s.replace(
-    ["noxfile.py"], "--cov=google", "--cov=sqlalchemy_bigquery",
+    ["noxfile.py"],
+    "--cov=google",
+    "--cov=sqlalchemy_bigquery",
 )
 
 
 s.replace(
-    ["noxfile.py"], 
+    ["noxfile.py"],
     "\+ SYSTEM_TEST_EXTRAS",
     "",
 )
 
 
 s.replace(
     ["noxfile.py"],
-    '''"protobuf",
-        # dependency of grpc''',
-    '''"protobuf",
-        "sqlalchemy<2.0.0",
-        # dependency of grpc''',
+    """"protobuf",
+        # dependency of grpc""",
+    """"protobuf",
+        "sqlalchemy>=1.4.16,<2.1",
+        # dependency of grpc""",
```

Comment:
I took a look at how Python handles a triple quote that encloses a double quote and while it might look weird, it *seems* to be successful.